app/src/services/react-query.js

```diff
@@ -2,7 +2,16 @@ import { capture } from "@/sentry";
 import { QueryCache, QueryClient } from "@tanstack/react-query";
 
 export const queryClient = new QueryClient({
-  queryCache: new QueryCache({ onError: (error) => capture(error) }),
+  queryCache: new QueryCache({
+    onError: (error) => {
+      if (error.message === "AuthenticationError") {
+        console.log("Unauthorized, reloading app");
+        window.location.reload(true);
```

Comment:
```suggestion
        if (window?.location?.pathname !== "/auth") {
            window.location.href = "/auth?disconnected=1";
            return;
        }
```

          On faisait ça historiquement

Reply:
qui derrière fait le API.setToken(null);

Reply:
Moi, j'avais l'impression qu'on pouvait simplifier pas mal ce bordel en passant toujours par le cookie et en arrêtant de faire le setToken

Reply:
Je suis d'accord, il y a un besoin de simplification. Il y a un micmac entre les setToken, les redirect to auth et les reloads. J'ai besoin de mieux comprendre comment ça marche.

Reply:
Hop token enlevés :)